### ISSUE TITLE
Long title while association iteration 

### ISSUE THREAD

**Issue by naina-verma** [fabric8io/fabric8-planner#1792]
Long title while association iteration 

- Paste very long title for the WI 
- while association it with Iteration you will see a bug scroll on

Video link: https://drive.google.com/a/redhat.com/file/d/0B1qUq_laG6COeGV1RDM3dnhFY00/view?usp=sharing

**Comment by naina-verma** [comment]
Hi @catrobson ,

can you please look into it if this issue needs some clarity in terms of UX designs for long titles !

**Comment by harrymower65** [comment]
Yes, we will need to truncate the work item name after so many characters.  I would try truncating the name after about 50 characters with an ellipsis.

**Comment by catrobson** [comment]
Yes, we should truncate the name to avoid this.  I would try to truncate after about 50 characters.

**Comment by SMahil** [comment]
@catrobson can we limit the title upto 255 characters as standard input field limit? Later on we can easily find if truncation is needed in modal as view perspective

**Comment by ldimaggi** [comment]
+1 on a consistent system-wide limit of 255 characters for one-line fields such as titles.
